### ISSUE TITLE
Three Black Crows possibly returning wrong result

### ISSUE THREAD

**Issue by KashyapCKotak** [anandanand84/technicalindicators#136]
Here is my input:
var input = {
  open: [7707.82, 7684.76, 7675.76],
  high: [7709.61, 7695.64, 7677.15],
  close: [7684.76, 7675.04, 7651.54],
  low: [7682.37, 7672.36, 7647.49]
}

Here's how it looks on a graph:
![image](https://user-images.githubusercontent.com/20476571/41138961-5f7a6d70-6b02-11e8-8593-879e3f04b4b9.png)

for three black crows:
var result = threeblackcrows(input);  //returns **FALSE**
shouldn't this return **true**??

**Comment by anandanand84** [comment]
seems correct to me.

**Comment by anandanand84** [comment]
The three black crows pattern should ideally consist of three relatively long bodied bearish candlesticks that close at or near the low price for the period. The second candle almost closes near the low

**Comment by KashyapCKotak** [comment]
aren't all the candles closing near low?

**Comment by anandanand84** [comment]
yes all candles should close near the low. Can you tell me what are you expecting in three black crows

**Comment by anandanand84** [comment]
ok got. It should return true instead it return false. May be it is also expecting high should be near the open.

**Comment by KashyapCKotak** [comment]
I tried reducing the middle one to have almost no upper shadow... But it still returned false...

**Comment by anandanand84** [comment]
ok I'll check it out.

**Comment by JMoli** [comment]
I see nothing wrong with the input / logic... If I had to guess I bet something is getting reversed before being passed to threeblackcrows... @KashyapCKotak Can you test the same input with the same implementation with threewhitesoldiers just to remove this as a possibility?

**Comment by JMoli** [comment]
Nvm I did it and see they are both false... Weird I will look closer

**Comment by JMoli** [comment]
Okay, the problem is the second day open is EQUAL to the first day close... looks like we need to add >= to the logic test instead of just greater than (edge case but needs to be handled)... but I am not sure what other implications this would bring...
